### ISSUE TITLE
Health endpoint

### ISSUE THREAD

**Issue by gmelodie** [application-research/estuary#252]
We should have a `/version` endpoint that returns the version an Estuary node and shuttle are running:

```
curl api.estuary.tech/health
{
    version: "version-here",
    ...
}
```

Also, have this data injested into Graphana

**Comment by en0ma** [comment]
@gmelodie you meant `/health` right? Also, I want to believe this is for uptime monitoring. If so, how do we intend to use it?

**Comment by gmelodie** [comment]
@en0ma we should document both the version and the health endpoints

**Comment by gmelodie** [comment]
I mean we should have those endpoints on https://docs.estuary.tech

**Comment by en0ma** [comment]
I do not think so, `/health` is for internal monitoring. If we want to allow users to know about API status - we need a status page for that, and its more than just a `health` endpoint

**Comment by gmelodie** [comment]
Still, the docs should exist for the team. Not sure where these would go though

**Comment by corinne-antonia** [comment]
I'll have @jimmylee added to the docs on estuary.tech and we can close this issue
